libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GSF405A/hwdef.dat

```diff
@@ -0,0 +1,165 @@
+# hw definition file for processing by chibios_pins.py
+# JHEMCU GSF405A by V-22
+# With F405 MCU, MPU6000 IMU and MAX7456 series OSD
+# Based on Mamba F405 from jeanphilippehell
+@ thanks to betaflight for pin information
+
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1060
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+# board voltage
+STM32_VDD 330U
+
+# need to use timer 4 so timer 5 is available for bidir DShot
+define STM32_ST_USE_TIMER 4
+define CH_CFG_ST_RESOLUTION 16
+
+# order of I2C buses
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+# this order follows the labels on the board
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 EMPTY USART6
+
+# The pins that USB is connected on
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USB detection
+PA8 VBUS INPUT OPENDRAIN
+
+# USART1 (DMA) - wired to on-board ELRS receiver - need DMA for CRSF protocol
+PB6 USART1_TX USART1
+PA10 USART1_RX USART1
+
+# default Serial1 to ELRS RX
+define HAL_SERIAL1_PROTOCOL 23
+define HAL_RSSI_TYPE 3 
+
+# Alt config to allow PPM on RX1
+#PA10 TIM1_CH3 TIM1 RCININT PULLDOWN ALT(1)
+
+# USART2
+PD5 USART2_TX USART2 NODMA
+PD6 USART2_RX USART2 NODMA
+
+# USART3 (DMA) - Use for GPS as it is close to I2C pins. 
+PB10 USART3_TX USART3
+PB11 USART3_RX USART3
+
+# UART4
+PA0 UART4_TX UART4 NODMA
+PA1 UART4_RX UART4 NODMA
+
+# USART6 (DMA)
+PC6 USART6_TX USART6
```

Comment:
You could probably drop DMA from here

Reply:
Does it negatively impact anything to have DMA here? We get it for "free" since there are less DMA conflicts on DMA2.

Reply:
If you get DMA for free then keep it.